### DISCUSSION TITLE
If we use the TUG products, is it necessary to estimate and correct the GNSS ACVs in POD?

### DISCUSSION THREAD

**Discussion by ParisaShafiei** [groops-devs/groops#79] [Q&A] [answered]
### Description

Dear all, 

Regarding the topic of my PhD research, I would like to use the Raw Observation Approach for the precise orbit determination purpose and then use those kinematic orbits for Earth's gravity field recovery. So, here I have some questions that I would be pleased if you could answer them. 

I have gone through on the latest article titles as "Improved precise kinematic LEO orbits based on the raw observation approach" of Graz University of Technology. In the article, it was mentioned that for doing POD, there are fine main and four essential steps which should be followed  to obtain the kinematic orbits as precisely as possible. However, there were some points which were not clear and then led to these two important question: 

**First:**

If I want to reprocess the kinematic orbits of some LEO missions like, GRACE Fo, Sentinel and Swarm to learning the software by doing some practical exercises. Thus, regarding the mentioned steps, should I reprocess the GNUS-Ground station network to determine the GNSS ACVs to obtain the correction patterns correct these products ? And then,  should I process estimate  the LEO ACVs to use them in the GNSS processing step and get the final kinematic orbits?  Indeed, when we use the TUG products in our orbit processing, is it necessary to determine the GNSS ACVs or not? 

**Second:** 
If I want to process another LEO mission which its metadata is not available on the Graz university's ftp page, should I follow the same steps as I have mentioned above to get the GNSS ACVs, correct the patterns, then use them to get the ACVs for the entire LEO mission so as to introduce them as known parameters in the POD processing and finally get the kinematic orbits? If it is right, how I can do it? 

Could you please tell me whether if there is any other reference explained all the details in these steps and how we can implement them in the GROOPS software? 

Like always, thank you in advance for your help and support. 

I am looking forward to hearing from you as soon as possible. 

With Best Wishes, 
Parisa 

### GROOPS version

Release 2021-09-06

### Operating systems

- [X] Linux
- [ ] Windows
- [ ] macOS
- [ ] Other (please specify in the description)

**Comment by bsuesser-rechberger** [comment, marked as answer]
Dear Parisa, 

For the kinematic orbit processing you can use our provided GNSS products, but these solutions only contain the official ANTEX R3 ACVs from IGS.
If you want to have your own estimated transmitter ACVs in the GNSS solution you have to calculate the network. (whereby this effort is not inconsiderable). An example scenario can be found in the cookbook and a detailed description of how the network is determined can be found in the PhD thesis by Sebastian Strasser.

Unfortunately, it is not possible for us to offer this kind of software support (to go through the kinematic orbit processing with you step by step), as we are only a small research group.

Best regards,
Barbara
